src/Catalog/Persistence/AzureCloudBlockBlob.cs

```diff
@@ -15,6 +16,12 @@ public sealed class AzureCloudBlockBlob : ICloudBlockBlob
     {
         private readonly CloudBlockBlob _blob;
 
+        public string ContentMD5
```

Comment:
Since for some reason it is string (I understand that that's what library provides), we need some documentation about what is exactly in the string.
Usually, string representation of a hash is a hex string, but the code below uses Base64 and the Azure .NET API [documentation](https://docs.microsoft.com/en-us/dotnet/api/microsoft.windowsazure.storage.blob.blobproperties.contentmd5?view=azure-dotnet#Microsoft_WindowsAzure_Storage_Blob_BlobProperties_ContentMD5) does not cover that.

Reply:
Added comment